games-util/pyfa/pyfa-2.28.1.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
Python 3.9 too, if tests pass?

Reply:
I will look into this once I get emerge to tell me what the actual issue is when it blocks the install. this is what I currently get:
```
emerge: there are no ebuilds to satisfy ">=dev-python/pyyaml-5.1[python_targets_python3_9(-)]".
(dependency required by "games-util/pyfa-2.28.1::devel[python_single_target_python3_9]" [ebuild])
(dependency required by "pyfa" [argument])
```
That ebuild does have python 3.9 support though :man_shrugging:

Reply:
There are at least two dependencies that have not yet the python 3.9 support. wxpython and logbook. Will make a PR for logbook.